test/integration/Region.e2e.test.js

```diff
@@ -4,100 +4,83 @@ describe('Regions', () => {
+            cy.showPreview(Cypress.env('FILE_ID_IMAGE'));
+
+            // Wait for the empty region layer to be present
+            cy.getByTestId('ba-Layer--region');
+
+            // Assert that the region creator does not exist and no annotations are present
+            cy.getByTestId('ba-RegionCreator').should('not.exist');
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('not.exist');
 
-    it('should hide region button for rotated image', () => {
-        // Show the preview
-        cy.showPreview(Cypress.env('FILE_ID_IMAGE'));
+            // Add a region annotation on the image
+            cy.addRegion();
+
+            // Assert that at least one annotation is present on the image
+            cy.get('[data-testid^="ba-AnnotationTarget"]');
+        });
+
+        it('should hide region button for rotated image', () => {
+            // Show the preview
+            cy.showPreview(Cypress.env('FILE_ID_IMAGE'));
+
+            // Assert region button is not hidden
+            cy.getByTestId('bp-AnnotationsControls-regionBtn').should('be.visible');
+
+            // Rotate image
+            cy.getByTitle('Rotate left').click();
+
+            // Assert region button is hidden
+            cy.getByTestId('bp-AnnotationsControls-regionBtn').should('not.be.visible');
+
+            // Rotate image back to non-rotated state
+            cy.getByTitle('Rotate left')
+                .click()
+                .click()
+                .click();
+
+            // Assert region button is not hidden
+            cy.getByTestId('bp-AnnotationsControls-regionBtn').should('be.visible');
+        });
+    });
 
-        // Assert region button is not hidden
-        cy.getByTestId('bp-AnnotationsControls-regionBtn').should('be.visible');
+    describe('Selecting region annotations', () => {
+        it('should select a region on a document', () => {
+            // Show the preview
+            cy.showPreview(Cypress.env('FILE_ID_DOC'));
 
-        // Rotate image
-        cy.getByTitle('Rotate left').click();
+            // Add a region annotation on the document
+            cy.addRegion();
 
-        // Assert region button is hidden
-        cy.getByTestId('bp-AnnotationsControls-regionBtn').should('not.be.visible');
+            // Assert that annotation target is not active
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('not.have.class', 'is-active');
 
-        // Rotate image back to non-rotated state
-        cy.getByTitle('Rotate left')
-            .click()
-            .click()
-            .click();
+            // Select annotation target
+            cy.get('[data-testid^="ba-AnnotationTarget"]').click();
 
-        // Assert region button is not hidden
-        cy.getByTestId('bp-AnnotationsControls-regionBtn').should('be.visible');
+            // Assert that annotation target is active
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('have.class', 'is-active');
```

Comment:
Should this be added to the existing "create" tests, instead? We're doing a lot of bootstrapping to test the presence of a class.